src/preact/base-element.js

```diff
@@ -373,6 +374,16 @@ export class PreactBaseElement extends AMP.BaseElement {
    */
   mutationObserverCallback() {}
 
+  /**
+   * A callback that allows the user to specify additional init properties
+   * to the Mutation Observer.
+   * @protected
+   * @return {!MutationObserverInit}
+   */
+  getAdditionalMutationObserverInitProperties() {
+    return {};
```

Comment:
Needed to add the additional init property `attributeOldValue` in Social Share.  Can also do this as a static class property.  Open to opinions, I put it here as a method so it can be next to the closely related `mutationObserverCallback()`